src/lib.rs

```diff
@@ -17,7 +17,7 @@
     clippy::disallowed_methods
 )]
 
-mod backoff;
+pub mod backoff;
```

Comment:
This may be excessive. If you want `BackoffConfig` to be available to users then expose that, this exposes all the internals of the backoff module

`pub use backoff::BackoffConfig` to just expose the config.

Reply:
Got it.